### DISCUSSION TITLE
Plan for generator-springboot 1.0.0 release

### DISCUSSION THREAD

**Discussion by sivaprasadreddy** [sivaprasadreddy/generator-springboot#122] [General]
I think it's time for **generator-springboot** to hit **1.0.0** version. I am thinking of coming up with a plan about what are the features we are going to add, update/remove any of the existing features, documentation updates etc.

- Upgrade to Spring Boot 3.2.0
- Add support for **Grafana Stack based monitoring support**
- **Revise/update ELK stack** to support newer version
- Streamline **"local development" setup using Testcontainers**
- Adding **mkdocs** based documentation

This is not a set in stone plan. We can add or remove tasks based on time availability.

Let me know your thoughts.

**Comment by rajadilipkolli** [comment]
I would like to add below features as well.

- [ ] Generating release notes based on commits, not sure if it is same as `Adding mkdocs based documentation`
- [ ] Automating pushing to npm
- [ ] Optional : Supporting Zipkin
- [x] Optional : provide framework for using DTO
- [ ] Optional : Incorporate best practices in spring boot series blog posts
